src/utils/window-console.js

```diff
@@ -90,6 +94,9 @@ export function logFriendlyPreamble() {
       %cwindow.filteredThread%c - The current filtered thread
       %cwindow.filteredMarkers%c - The current filtered and processed markers
       %cwindow.callTree%c - The call tree of the current filtered thread
+      %cwindow.getState%c - The function that returns the current Redux state.
+      %cwindow.dispatch%c - The function to dispatch a Redux action to change the state.
+      %cwindow.selectors%c - All the selectors that are used to get a data from the Redux state.
```

Comment:
```suggestion
      %cwindow.selectors%c - All the selectors that are used to get data from the Redux state.
```

nit: `data` is a plural world